railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_5_2.rb.tt

```diff
@@ -9,3 +9,12 @@
 # Make Active Record use stable #cache_key alongside new #cache_version method.
 # This is needed for recyclable cache keys.
 # Rails.application.config.active_record.cache_versioning = true
+
+# Use AES 256 GCM authenticated encryption for encrypted cookies.
+# Existing cookies will be converted on read then written with the new scheme.
+Rails.application.config.action_dispatch.use_authenticated_cookie_encryption = false
+
+# No longer upgrade CBC HMAC legacy encrypted cookies. This should be disabled
+# when this legacy form of encrption no longer needs to automatically upgraded
+# to GCM encryption.
+# Rails.application.config.action_dispatch.upgrade_legacy_hmac_aes_cbc_cookies = false
```

Comment:
See my [latest comment](https://github.com/rails/rails/pull/28132#issuecomment-302944890) for more discussion on this flag. I'm thinking to we just leave the upgrade behavior as is and have future version use "pure" GCM cookies...